WWW/1_first_task.wdl

```diff
@@ -15,44 +15,29 @@ workflow minidata_test_alignment {
   input {
     # Sample info
     File sampleFastq
-    String base_file_name
     # Reference Genome information
-    String ref_name
     File ref_fasta
     File ref_fasta_index
     File ref_dict
-    # This is the .alt file from bwa-kit (https://github.com/lh3/bwa/tree/master/bwakit),
-    # listing the reference contigs that are "alternative". Leave blank in JSON for legacy
-    # references such as b37 and hg19.
-    File? ref_alt
     File ref_amb
     File ref_ann
     File ref_bwt
     File ref_pac
     File ref_sa
   }
 
-    # Docker containers this workflow has been designed for
-    String bwadocker = "fredhutch/bwa:0.7.17"
-    String GATKdocker = "broadinstitute/gatk:4.1.4.0"
-    
-    #Array[Object] batchInfo = read_objects(batchFile)
-
   #  Map reads to reference
   call BwaMem {
     input:
       input_fastq = sampleFastq,
-      base_file_name = base_file_name,
       ref_fasta = ref_fasta,
       ref_fasta_index = ref_fasta_index,
       ref_dict = ref_dict,
-      ref_alt = ref_alt,
       ref_amb = ref_amb,
       ref_ann = ref_ann,
       ref_bwt = ref_bwt,
       ref_pac = ref_pac,
-      ref_sa = ref_sa,
-      taskDocker = bwadocker
```

Comment:
ref_amb is a file --> ~{ref_amb} 

`mv ~{ref_amb} .`

note to ash: look more into how miniwdl vs cromwell localizes inputs and where

Reply:
[Cromwell on HPC:](https://cromwell.readthedocs.io/en/stable/backends/HPC/) 
> Any input files to a call need to be localized into the <call_dir>/inputs directory. 

(did a quick search for miniwdl, didn't find anything at first.)

Reply:
If I run a workflow on miniwdl locally where, in the command section, I echo the input file, the output I get is this:
`/mnt/miniwdl_task_container/work/_miniwdl_inputs/0/10_SRA_samples.pb`

`pwd` returns:
`/mnt/miniwdl_task_container/work`